spec.bs

```diff
@@ -3269,28 +3269,22 @@ To <dfn>handle server response private aggregation fields</dfn> given a [=server
 
 <div algorithm>
 To <dfn>commit server response private aggregation contributions</dfn> given a [=map=] from a
-[=tuple=] of ([=reporting bid key=], [=string=]) to a [=list=] of [=on event contribution entries=]
+[=server auction pagg contribution key=] to a [=list=] of [=on event contribution entries=]

```

Comment:
The "pagg" in the same is kinda awkward...

Reply:
changed to PAgg

Reply:
I think it would probably be best to just write it out as "private aggregation", even if it makes it quite long.

Reply:
done.